src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -157,22 +159,39 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
         return Promise.resolve(trialJobOutputUrl);
     }
     
+    /**
+     * generate trial's command for frameworkcontroller
+     * expose port and execute injector.sh before user's command
```

Comment:
before **executing** user's command

Reply:
updated to 'before executing user's command'